src/utils/ApiUtils.js

```diff
@@ -1,3 +1,5 @@
+export const weatherbaseUrl = 'https://query.yahooapis.com/v1/public/yql';
```

Comment:
I think Garry's PR has a config object. We'll move this URL in there when that's in.